testing/cypress.config.js

```diff
@@ -14,7 +14,22 @@ reporter: 'cypress-mochawesome-reporter',
     chromeWebSecurity: false,
     responsetimeout: 10000,
     "blockHosts": ["www.google-analytics.com", "ssl.google-analytics.com"],
-    
+
+    env: {
+      // info for site admin role
+      "site_admin": "admin_test",
+      "admin_password": "vote123",
+      "admin_role": "site_admin",
+      // info for content editor role
+      "content_editor": "editor_test",
+      "editor_password": "vote1234",
+      "editor_role": "content_editor",
+      // info for content mananger role 
+      "content_manager": "manager_test",
+      "manager_password": "vote123",
+      "manager_role": "content_manager",
+    },
```

Comment:
Update the object to look like this. A single common password for all roles. Then update the tests to grab the values from the object.
```
env: {
      "test_pass": "password",
      "roles": {
            // info for site admin role
            "site_admin": {
                  "username": "admin_test",
                  "role": "site_admin"
            },
            // info for content editor role
            "content_editor": {
                  "username": "editor_test",
                  "role": "content_editor"
            },
            // info for content mananger role 
            "content_manager": {
                  "username": "manager_test",
                  "role": "content_manager"
            }
      }
},
```

Reply:
updated env format